deck/templates/proposal/my_proposals.html

```diff
@@ -0,0 +1,52 @@
+{% extends "layout.html" %}
+
+{% load i18n %}
+{% load bootstrap3 %}
+{% load deck_tags %}
+
+{% block head_title %}{% trans "My Proposals" %}{% endblock %}
+
+{% block content %}
+  <div class="row" id="event-detail">
```

Comment:
Put this code, i think it will keep the layout more likelly the event list page.

o/

```
  <div class="row header">
    <h2><strong>{% trans "My Proposals" %}</strong></h2>
  </div>

  <div class="row" id="proposal-list">
```